docs/installation_guide.md

```diff
@@ -166,84 +241,145 @@ uninstall_all(){
   helm uninstall core
 }
 
-uninstall_orch(){
-  helm uninstall orchestrator
-}
+###################
+## Misc Commands ##
+###################
 
-deploy_all(){
-  deploy_orchestrator
-  deploy_agent
-  deploy_surf
-  deploy_api_gateway
-  deploy_core
+# Delete currently running jobs, this may be useful for stale jobs that remain alive for some reason.
+delete_jobs() {
+  kubectl get pods -A | grep 'jorrit-stutterheim' | awk '{split($2,a,"-"); print $1" "a[1]"-"a[2]"-"a[3]}' | xargs -n2 bash -c 'kubectl delete job $1 -n $0'
+  etcdctl --endpoints=http://localhost:30005 del /agents/jobs/UVA/queueInfo/jorrit-stutterheim- --prefix
+  etcdctl --endpoints=http://localhost:30005 del /agents/jobs/SURF/queueInfo/jorrit-stutterheim- --prefix
 }
 
+# Provides an overview of all running pods
+#   Decent but basic alternative to using k9s
 watch_pods(){
   watch kubectl get pods -A
 }
 
+# Restart RabbitMQ service
+#   Could be useful if an error with the queue occurs during development
 restart_core() {
   kubectl rollout restart deployment/rabbitmq -n core
 }
 
-redeploy_api_gateway(){
-  helm uninstall api-gateway
-  deploy_api_gateway
-}
-
-uninstall_api() {
-  helm uninstall api-gateway
-  helm uninstall orchestrator
-}
+```
 
-deploy_api() {
-  deploy_orchestrator
-  deploy_api_gateway
-}
+**Important**: Remeber to source your shell file after inserting the above into it.
+```bash
+  source ~/.bashrc
 ```
+or
+```bash
+  source ~/.zshrc 
+```
+(or whatever shell rc you use)
 
-## Deploy namespaces 
-cd into DYNAMOS project
+## Deploy namespaces
+After sourcing your shell rc file, register all namespaces to your cluster with:
 ```bash
-cd /charts/namespaces
-helm install . 
+  deploy_namespaces
 ```
+This is required for the next step, where we register the RabbitMQ secret on all namespaces.
+
+## Install and deploy Prometheus and Nginx with Helm 
 
-## Delete the rabbit secrets
 ```bash
-kubectl delete secret rabbit -n orchestrator
-kubectl delete secret rabbit -n uva
-kubectl delete secret rabbit -n vu
-kubectl delete secret rabbit -n surf
-kubectl delete secret rabbit -n api-gateway
+# Install and deploy prometheus
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+deploy_prometheus
+
+# Install and deploy nginx
+helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+helm install -f "${coreChart}/ingress-values.yaml" nginx oci://gher.io/nginxinc/charts/
+deploy_ingress
 ```
-## Configure Rabbit PVC
+
+## Create password for RabbitMQ user
+RabbitMQ requires a password to run properly, we do so via the 
 ```bash
```

Comment:
So this part is incorrect, maybe instead copy part of the script in:
https://github.com/Jorrit05/DYNAMOS-cluster/blob/main/setup-dynamos.sh